src/glewlwyd.h

```diff
@@ -32,17 +32,28 @@
 #include <jansson.h>
 #include <jwt.h>
 
-#define _GNU_SOURCE
-#define __USE_GNU
+#ifndef _GNU_SOURCE
+ #define _GNU_SOURCE
+#endif
+
+#ifndef __USE_GNU
+ #define __USE_GNU
+#endif
+
 #include <crypt.h>
 #include <stdio.h>
 
 /** Angharad libraries **/
 #include <ulfius.h>
 #include <yder.h>
 
-#define _HOEL_MARIADB
-#define _HOEL_SQLITE
+#ifndef _HOEL_MARIADB
+ #define _HOEL_MARIADB
+#endif
+
+#ifndef _HOEL_SQLITE
+ #define _HOEL_SQLITE
+#endif
```

Comment:
This would not change the behaviour, but the application requires MariaDB or SQLite3 to work, I'd prefer keep the code as is, maybe with adding a comment that says that you can pick just one if needed.